### PR TITLE
[Subtask][KPIP-5] Fix npm run build fail with js

### PR DESCRIPTION
**Pull request by zwangsheng**
### _Why are the changes needed?_

`npm run build` fail with following log
```
error TS6504: File '/Users/yangbinjie01/IdeaProjects/incubator-kyuubi/kyuubi-server/web-ui/src/views/operation/completedJobs/index.vue.__VLS_script.js' is a JavaScript file. Did you mean to enable the 'allowJs' option?
  The file is in the program because:
    Root file specified for compilation
...
```
### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [x] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.apache.org/docs/latest/develop_tools/testing.html#running-tests) locally before make a pull request

WX20220825-150028

### CODE REVIEW

**Comment by zwangsheng** [comment]
cc @pan3793 @tongwl

**Comment by tongwl** [comment]
@zwangsheng 
Hi, please also change  "dev": "vite --port 9090" to  "dev": "vue-tsc --noEmit && vite --port 9090" in package.json, expose the problem of missing "lang=ts" in the dev stage instead of the build stage.

**Comment by zwangsheng** [comment]
> @zwangsheng Hi, please also change "dev": "vite --port 9090" to "dev": "vue-tsc --noEmit && vite --port 9090" in package.json, expose the problem of missing "lang=ts" in the dev stage instead of the build stage.

OK, nice catch. + 1 for throw exception during dev.

**Review by tongwl — APPROVED** [review]

File: kyuubi-server/web-ui/src/views/contact/index.vue [hunk 1]
```diff
@@ -20,7 +20,7 @@
   <main>Contact page</main>
 </template>
 
-<script>
+<script lang='ts'>
```
Comment (pan3793) [inline kyuubi-server/web-ui/src/views/contact/index.vue:23]: nit: "double qoute"

**Review by pan3793 — APPROVED** [review]

**Comment by pan3793** [comment]
Thanks, merging to master

**Pull request closed without merging**